images/openshift-base-nodejs.rhel9.yml

```diff
@@ -0,0 +1,35 @@
+base_only: true
+content:
+  source:
+    git:
+      branch:
+        target: openshift-base-nodejs-rhel9
```

Comment:
@joepvd I'm guessing we can try with `openshift-base-nodejs` now right?

Reply:
No. Let's keep those separate. One source flowing into multiple distinct items is something to avoid, imho.